cardano-sl/cardano-sl.cabal

```diff
@@ -3,13 +3,13 @@ version:             0.5.1
 synopsis:            Cardano SL main implementation
 description:         Please see README.md
 license:             MIT
-license-file:        LICENSE
+license-file:        ../LICENSE
```

Comment:
You'll need to move files to subfolder as well and then symlink them at the top-level, otherwise Nix won't work properlly